pkg/workloads/cortex/tf_api/api.py

```diff
@@ -93,6 +94,20 @@
 }
 
 
+@app.after_request
+def after_request(response):
```

Comment:
In the onnx file, this was merged with the request logging. Can you either merge here or split there?